frontend/src/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import '~bootstrap/scss/bootstrap.scss';
+@import '../node_modules/bootstrap/scss/bootstrap.scss';
```

Comment:
Is this the best approach? It looks a bit strange.

Reply:
The `~` is deprecated. I don't like it either, but I don't know any better alternatives...